modules/default.nix

```diff
@@ -6,6 +6,8 @@
 , check ? true
   # Extra arguments passed to specialArgs.
 , extraSpecialArgs ? { }
+  # If disabled, the pkgs attribute passed to this function is used instead.
+, useNixpkgsModule ? true
```

Comment:
I agree that setting this to `true` is the better choice, but I am sure there will be bug tickets related to this change.